src/langauge-servers/goUtils.ts

```diff
@@ -69,9 +69,12 @@ const fromCommonPlaces = (): string[] => {
         });
     } else if (isLinux || isMac) {
         // common place for Linux and macOS
-        ["/usr/local/bin", "/usr/bin"].forEach(goLocation => {
+        const stdout = cp.execSync("go env GOROOT", { encoding: "utf-8", stdio: ["inherit"] });
+        const locations = _.compact(_.split(stdout, /[\n\r]+/));
```

Comment:
For readability reasons only I would place the `\r` before the `\n` :
`const locations = _.compact(_.split(stdout, /[\r\n]+/));`

Reply:
> For readability reasons only I would place the `\r` before the `\n` : `const locations = _.compact(_.split(stdout, /[\r\n]+/));`

@EliRookout Done